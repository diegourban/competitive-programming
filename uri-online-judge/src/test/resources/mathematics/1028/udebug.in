10
1000 190
345 211
90 21
220 1000
1000 1000
2 2
1 1
200 300
120 300
450 600