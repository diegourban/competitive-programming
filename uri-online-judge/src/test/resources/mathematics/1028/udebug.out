10
1
3
20
1000
2
1
100
60
150
